### PR TITLE
Web UI: Show a context help about storage options

### PR DESCRIPTION
**Pull request by poncovka**
Add a button for showing text about the storage options.

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me except for a small detail. Thank you!

The code is what it is, unfortunately. Given that the help drawer would eventually be available in other screens too, it seems necessary to have it wrap the wizard, even if that seems backward.

File: ui/webui/src/components/storage/InstallationDestination.jsx [hunk 1]
```diff
@@ -267,15 +267,22 @@ export const InstallationDestination = ({ idPrefix, onAddErrorNotification }) =>
...
-                cockpit.format(_(
+            <Text component={TextVariants.p}>
+                {cockpit.format(_(
                     "Select the device(s) to install to. The installation requires " +
-                    "$0 of available space. Storage will be automatically partitioned."
-                ), cockpit.format_bytes(requiredSize))
-            }
+                    "$0 of available space. Storage will be automatically partitioned. "
+                ), cockpit.format_bytes(requiredSize))}
+                <Button variant="link" isInline onClick={onClick}>
```
[the remark this answers is not in the document]
Comment (poncovka) [reply ui/webui/src/components/storage/InstallationDestination.jsx]: Good point. Fixed.

**Comment by poncovka** [comment]
> The code is what it is, unfortunately. Given that the help drawer would eventually be available in other screens too, it seems necessary to have it wrap the wizard, even if that seems backward.

Btw. the drawer covers the whole screen. That's the reason why this component has to be defined on the top level.

**Comment by poncovka** [comment]
Updated.

**Comment by poncovka** [comment]
All `assert_pixels` calls are failing on a condition that checks running animations (https://github.com/cockpit-project/cockpit/blob/3d8af4cd125293c5c258c48209606a258a22d1a7/test/common/testlib.py#L866-L887). It looks like the drawer creates an infinite animation and Cockpit doesn't offer any support for handling that. @M4rtinK, what do you suggest to do?

**Comment by M4rtinK** [comment]
> All `assert_pixels` calls are failing on a condition that checks running animations (https://github.com/cockpit-project/cockpit/blob/3d8af4cd125293c5c258c48209606a258a22d1a7/test/common/testlib.py#L866-L887). It looks like the drawer creates an infinite animation and Cockpit doesn't offer any support for handling that. @M4rtinK, what do you suggest to do?

I'll look into it - if we can't resolve it somewhat quickly, I guess we could include this out-of-band just in the Web UI testing image until the issues are resolved.

**Comment by VladimirSlavik** [comment]
Can we just not test the drawer for now? Alternatively, can we maybe test that uf the drawer is out, the assert times out?

**Comment by poncovka** [comment]
> Can we just not test the drawer for now? Alternatively, can we maybe test that uf the drawer is out, the assert times out?

I don't test the drawer at all. The drawer is part of the screen even when it is not visible and it messes up other tests. We would have to disable all pixel tests we have to make it work.

**Comment by VladimirSlavik** [comment]
Oh my, that's hilarious.

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :) Strange thing with the "eternal animation" but I'm sure we will be able to track it down eventually. 

@martinpitt do you use the Drawer component in cockpit & have you spotted some pixel test issues with it ?

**Comment by M4rtinK** [comment]
Rebased PR with text from Allison:
Snímek obrazovky 2022-06-09 v 22 51 54

**Comment by martinpitt** [comment]
No, we don't use [drawer](https://www.patternfly.org/v4/components/drawer/) anywhere. Not sure what an "infinite animation" is, but if that can be detected, this should be fixed in testlib?

**Comment by poncovka** [comment]
The placeholder text was replaced with a real content.

**Comment by poncovka** [comment]
@martinpitt There seems to be this `pf-remove-tab-focus` animation that never ends when the drawer is not expanded.
Screenshot from 2022-06-16 19-00-26

**Comment by M4rtinK** [comment]
> No, we don't use [drawer](https://www.patternfly.org/v4/components/drawer/) anywhere. Not sure what an "infinite animation" is, but if that can be detected, this should be fixed in testlib?

@martinpitt Yeah, I think it something that will have to be fixed in the Cockpit testlib - just in case I've rebased tis pr on top of our upstream branch and re-run the tests, but its still the same:

`wait_js_cond(ph_count_animations("#app") == 0): Uncaught (in promise) Error: condition did not become true`

Until this is fixed in testlib I'm afraid we are blocked on this PR & any follow in-context help work.

**Comment by poncovka** [comment]
The tests are working.

**Pull request merged**